R/dessert.R

```diff
@@ -0,0 +1,26 @@
+#' Run a dessert recipe on an analysis or data set
+#'
+#' @param data The input data set or fit object to call dessert on.
+#' @param output_format The R markdown output format parsed to `rmarkdown`.
+#' @param output_dir The output directory for the rendered output files.
+#'
+#' @return A reproducible standard publication archive
+#' @export
+#'
+#' @examples
+#' fit <- lm(dist ~ speed, data = cars)
+#' dessert(fit)
+dessert <- function(data,
+                    output_format = NULL,
```

Comment:
Just checking if this is the right naming convention for the function arguments in the entire package?

Reply:
maybe we'll rename `data` to `input` as a more generic term.